libraries/Microsoft.Bot.Builder.Dialogs/Internal/DateTimePromptInternal.cs

```diff
@@ -44,26 +44,21 @@ protected DateTimePromptInternal(IModel model, PromptValidator<DateTimeResult> v
                 var results = _model.Parse(message.Text);
                 if (results.Any())
                 {
-                    var result = results.First();
-                    if (result.Resolution.Any())
+                    var values = (List<Dictionary<string, string>>)results[0].Resolution["values"];
```

Comment:
Are we still dropping results that might have relevant values? (Can't there be 0,1,2, or 4 results? Or, am I thinking of values per result?